### PR TITLE
feat(newserver): Core Keeper

### LINKED ISSUE #3855: [Server Request] - Core Keeper
## What game server would you like to add?

Core Keeper

## Can you link to any tutorials or guides?

(https://steamdb.info/app/1963720/config/)

## Is the server on Steam?

* [X] Yes
* [ ] No

If yes Use SteamDB to get the appid. (https://steamdb.info).

 1963720

## Is the server supported on Linux?

We only support Linux servers and do **not** support Wine.

* [X] Yes
* [ ] No

### PR DESCRIPTION
**Pull request by h3o66**
# Description

Add Server Core Keeper

Fixes #3855

## Type of change

* [ ] Bug fix (a change which fixes an issue).
* [ ] New feature (change which adds functionality).
* [x] New Server (new server added).
* [ ] Refactor (restructures existing code).
* [ ] Comment update (typo, spelling, explanation, examples, etc).

## Checklist

PR will not be merged until all steps are complete.

* [x] This pull request links to an issue.
* [x] This pull request uses the `develop` branch as its base.
* [x] This pull request Subject follows the Conventional Commits standard.
* [x] This code follows the style guidelines of this project.
* [x] I have performed a self-review of my code.
* [x] I have checked that this code is commented where required.
* [x] I have provided a detailed with enough description of this PR.
* [x] I have checked If documentation needs updating.

## Documentation

If documentation does need updating either update it by creating a PR (preferred) or request a documentation update.
* User docs: https://github.com/GameServerManagers/LinuxGSM-Docs
* Dev docs: https://github.com/GameServerManagers/LinuxGSM-Dev-Docs

**Thank you for your Pull Request!**

### CODE REVIEW

**Comment by Panzke** [comment]
Since a short time the following error appears when running "update-lgsm":

checking GitHub module lgsm...ERROR
removing module lgsm...rm: cannot remove 'lgsm': Is a directory FAIL

File: lgsm/config-default/config-lgsm/ckserver/_default.cfg [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+##################################
+# DO NOT EDIT, ANY CHANGES WILL BE OVERWRITTEN!
+# Copy settings from here and use them in either:
+# common.cfg - applies settings to every instance.
+# [instance].cfg - applies settings to a specific instance.
+
+#### Game Server Settings ####
+
+## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
+# Edit with care | https://colonysurvival.gamepedia.com/Dedicated_Server#Installation_.28Linux.29
```
Comment (Chi-Yu) [inline lgsm/config-default/config-lgsm/ckserver/_default.cfg:12]: This line still references colony survival and should probably be updated or removed accordingly.

Comment (h3o66) [reply lgsm/config-default/config-lgsm/ckserver/_default.cfg:12]: Thanks, will address this with the next rebase

**Comment by h3o66** [comment]
> Since a short time the following error appears when running "update-lgsm":
> 
> checking GitHub module lgsm...ERROR removing module lgsm...rm: cannot remove 'lgsm': Is a directory FAIL

strange - did not have this one time - updated the pr now to the latest release of lgsm

**Comment by h3o66** [comment]
Latest changes did add xvfb and the log setting to the PR

**Review by dgibbs64 — APPROVED** [review]

**Pull request merged**